sdk/web-pubsub/web-pubsub/review/web-pubsub.api.md

```diff
@@ -0,0 +1,188 @@
+    readonly apiVersion: string;
+    readonly endpoint: string;
+    readonly groupName: string;
+    hasUser(username: string, options?: GroupHasUserOptions): Promise<boolean>;
+    readonly hubName: string;
+    removeConnection(connectionId: string, options?: GroupRemoveConnectionOptions): Promise<RestResponse>;
+    removeUser(username: string, options?: GroupRemoveUserOptions): Promise<RestResponse>;
+    sendToAll(message: string, options: GroupSendTextToAllOptions): Promise<RestResponse>;
+    sendToAll(message: JSONTypes, options?: GroupSendToAllOptions): Promise<RestResponse>;
+    sendToAll(message: HttpRequestBody, options?: GroupSendToAllOptions): Promise<RestResponse>;
+    // (undocumented)
+    sendToAll(message: string | HttpRequestBody, options?: GroupSendToAllOptions | GroupSendTextToAllOptions): Promise<RestResponse>;
+}
+
+// @public
+export class WebPubSubServiceClient {
+    constructor(connectionString: string, hubName: string, options?: HubAdminClientOptions);
+    constructor(endpoint: string, credential: AzureKeyCredential, hubName: string, options?: HubAdminClientOptions);
+    readonly apiVersion: string;
+    closeConnection(connectionId: string, options?: CloseConnectionOptions): Promise<RestResponse>;
+    endpoint: string;
+    getAuthenticationToken(options?: GetAuthenticationTokenOptions): Promise<GetAuthenticationTokenResponse>;
+    // (undocumented)
+    grantPermission(connectionId: string, permission: Permission, options?: HubGrantPermissionOptions): Promise<RestResponse>;
+    group(groupName: string): WebPubSubGroup;
+    hasConnection(connectionId: string, options?: HasConnectionOptions): Promise<boolean>;
+    hasGroup(groupName: string, options?: HubHasGroupOptions): Promise<boolean>;
+    hasUser(username: string, options?: HubHasUserOptions): Promise<boolean>;
+    readonly hubName: string;
+    removeUserFromAllGroups(userId: string, options?: CloseConnectionOptions): Promise<RestResponse>;
+    // (undocumented)
+    revokePermission(connectionId: string, permission: Permission, options?: HubRevokePermissionOptions): Promise<RestResponse>;
+    sendToAll(message: string, options: HubSendTextToAllOptions): Promise<RestResponse>;
+    sendToAll(message: JSONTypes, options?: HubSendToAllOptions): Promise<RestResponse>;
+    sendToAll(message: HttpRequestBody, options?: HubSendToAllOptions): Promise<RestResponse>;
+    sendToConnection(connectionId: string, message: string, options: HubSendTextToConnectionOptions): Promise<RestResponse>;
+    sendToConnection(connectionId: string, message: JSONTypes, options?: HubSendToConnectionOptions): Promise<RestResponse>;
+    sendToConnection(connectionId: string, message: HttpRequestBody | JSONTypes, options?: HubSendToConnectionOptions | HubSendTextToConnectionOptions): Promise<RestResponse>;
+    sendToUser(username: string, message: string, options: HubSendTextToUserOptions): Promise<RestResponse>;
```

Comment:
same thought here about making message not require a special option when it's `text/plain`